src/pg_query_fingerprint_defs.c

```diff
@@ -11639,6 +11602,13 @@ _fingerprintTableLikeClause(FingerprintContext *ctx, const TableLikeClause *node
     XXH3_freeState(prev);
   }
 
+  if (node->relationOid != 0) {
+    char buffer[50];
+    sprintf(buffer, "%d", node->relationOid);
+    _fingerprintString(ctx, "relationOid");
+    _fingerprintString(ctx, buffer);
+  }
```

Comment:
This adds the relation oid to the fingerprint if non-zero? Is this part of one of the changes in the description?

Reply:
Hm, interesting. I wonder how that got in there. That seems more by accident than intentional. Will review (and likely remove). Thanks for noticing!

Reply:
Upon further review, this looks like it was missed in a prior update. The relationOid is always 0 in the raw parser, so this is effectively a no-op.